UnleashedRecomp/locale/locale.cpp

```diff
@@ -703,6 +703,17 @@ std::unordered_map<std::string_view, std::unordered_map<ELanguage, std::string>>
             { ELanguage::Italian,  "Impossibile trovare il modulo \"%s\".\n\nAssicurati che:\n\n- Hai estratto questa copia di Unleashed Recompiled correttamente e non solo il file *.exe.\n- Non stai eseguendo Unleashed Recompiled da un file *.zip." }
         }
     },
+    {
+        "System_MemoryAllocationFailed",
+        {
+            { ELanguage::English,  "Memory allocation failed blablabla enable your pagefile, ensure it's minimum 4 or 8 GB blablablabla" },
```

Comment:
English text proposal:

---

Failed to allocate game memory.

Please make sure that:

- You meet the minimum system requirements (8 GB).
- Your pagefile is configured with at least 4-8 GB of virtual memory.

---

```suggestion
            { ELanguage::English,  "Failed to allocate game memory.\n\nPlease make sure that:\n\n- You meet the minimum system requirements (8 GB).\n- Your pagefile is configured with at least 4-8 GB of virtual memory." },
```